commands/root.go

```diff
@@ -72,6 +76,27 @@ Complete documentation is available at https://kool.dev/docs`,
 				hasWarnedDevelopmentVersion = true
 			}
 		},
+		RunE: func(cmd *cobra.Command, args []string) (err error) {
+			if len(args) == 0 {
+				err = cmd.Help()
+				return
+			}
+
+			scriptParser := parser.NewParser()
+
+			// look for kool.yml on current working directory
+			_ = scriptParser.AddLookupPath(environment.NewEnvStorage().Get("PWD"))
```

Comment:
The `run` command also looks for scripts inside `$HOME/kool` folder
https://github.com/kool-dev/kool/blob/1ed97f479e099fb13f30dd5b3ba57b55f50b801b/commands/run.go#L69